src/com_tjvendors/admin/sql/updates/mysql/1.4.3.sql

```diff
@@ -0,0 +1,33 @@
+ALTER TABLE `#__tjvendors_passbook` CHANGE `vendor_id` `vendor_id` INT(11)  NOT NULL DEFAULT 0;
+ALTER TABLE `#__tjvendors_passbook` CHANGE `currency` `currency` VARCHAR(255)  NOT NULL DEFAULT '';
```

Comment:
can we not mark column as NULLable instead of setting default as blank? applies to all such cols

Reply:
This is now Joomla sets default value for varchar type of column